13.technical-support/02.Adware/docs.en.md

```diff
@@ -0,0 +1,84 @@
+
+## How to clean your system from Adware?
+
+In the Internet, you can find a lot of applications that are aimed at fighting this problem. You can use any utility you like. But we strongly recommend using **[AdwareCleaner](https://downloads.malwarebytes.com/file/adwcleaner)** from **Malwarebytes**. This application is quite easy to use and you can test it by yourself.
+
+After the application has been downloaded and successfully installed, you should take the following steps:
+
+1) Start the application and tap **scan now**.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware1.png" />
+
+2) Once the scan has been completed, you will need to send suspicious objects to **quarantine**.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware2.png" />
+
+3) When the items are moved to **quarantine**, you will be able to **clear the system** of unnecessary ballast.
+4) In some cases, you may need to **reboot the system** after cleaning is complete.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware3.png" />
+
+5) Once the system has been rebooted, cleaning will be completed, if necessary, you can always read the **report** on the work done.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/adware4.png" />
+
+If it didn’t help - disable all settings in the browser in which this situation occurred. Use [this instructions](#instruction). 
+
+If disabling of settings and extensions does not help, try to [re-create shortcuts of your browsers](#shortcuts).
+
+Below on this page we’ve described the instructions for removing some of the most popular ad extensions.
+
+<a id="instruction"></a>
+### How to disable settings and extensions in the browser? 
+
+Below are the instructions on how to disable settings and extensions for the most popular browsers.
+
+If you use Google Chrome, [read this manual](https://support.google.com/chrome/answer/187443?hl=en).
+
+If you use Mozilla Firefox, [read this manual](https://support.mozilla.org/en-US/kb/disable-or-remove-add-ons).
+
+If you use the browser Opera, [read this manual](http://help.opera.com/Windows/11.50/en/extensions.html).
+
+<a id="shortcuts"></a>
+### How to re-create shortcuts of your browsers?
+
+Very often, malware and adware programs install you a homepage in a very simple way - they substitute your browser shortcut on the desktop, taskbar and "Start" menu, replacing it with the advertising link.
+
+The simplest solution in this case is to manually delete all the shortcuts and recreate them by yourself.
+
+### How to remove sindex.biz?
+
+If instead of the home page, you always open sindex.biz, it’s most likely there was a substitution of shortcuts of your browser. Try to [recreate the shortcuts](#shortcuts) to get rid of this problem.
```

Comment:
If instead of the home page, you always open sindex.biz, it’s likely that shortcuts for your browser(s) have been substituted. Try to [recreate the shortcuts](#shortcuts) to get rid of this problem.